### PR TITLE
Added profile for Creality CR-6 SE and CR-6 Max

### PR DESCRIPTION
**Pull request by mens**
This adds support for Creality CR-6 SE and CR-6 Max. It's based on the profiles in Bambu Slicer, but modified to be more inline with the existing Creality profiles in OrcaSlicer.

### CODE REVIEW

File: resources/profiles/Creality.json [hunk 1]
```diff
@@ -384,6 +603,41 @@
...
+			"name": "Creality CR-6 Max 0.6 nozzle",
+			"sub_path": "machine/Creality CR-6 Max 0.6 nozzle.json"
+		},
+		{
+			"name": "Creality CR-6 Max 0.8 nozzle",
+			"sub_path": "machine/Creality CR-6 Max 0.8 nozzle.json"
+		},
+		{
+=======
+>>>>>>> main
```
Comment (SoftFever) [inline resources/profiles/Creality.json:640]: errors

File: resources/profiles/Creality.json [hunk 2]
```diff
@@ -196,6 +351,70 @@
...
+			"name": "0.28mm SuperDraft @Creality 0.6 CR-6",
+			"sub_path": "process/0.28mm SuperDraft @Creality 0.6 CR-6.json"
+		},
+		{
+			"name": "0.20mm Standard @Creality CR-6 0.6",
+			"sub_path": "process/0.20mm Standard @Creality CR-6 0.6.json"
+		},
+		{
+			{
+=======
```
Comment (SoftFever) [inline resources/profiles/Creality.json:416]: Seems not merged properly

**Comment by SoftFever** [comment]
Hi @mens 
Thanks for the PR.
Seems it's not merged properly.

**Comment by mens** [comment]
I removed the MC artifacts. Sorry for those.

**Comment by SoftFever** [comment]
Hi @mens 
it's still not working.
With changes proposed in this PR, Creality profiles won't be able to able anymore.
Please double check

**Comment by mens** [comment]
Hi, I know, I'm reworking the whole thing.

**Comment by mens** [comment]
Please review again

**Review by SoftFever — APPROVED** [review]
Look good

**Comment by SoftFever** [comment]
Thanks for putting time to rework it.
I appreciate it 👍

**Pull request merged**